src/slurmdbd_ops.py

```diff
@@ -0,0 +1,447 @@
+#!/usr/bin/env python3
```

Comment:
[nit]: Since this is a imported module, it doesn't need to have a hashbag at the top of the file

```suggestion
```